### ISSUE TITLE
Using submodules cause Git Seekrets to output confusing error message.

### ISSUE THREAD

**Issue by rogeruiz** [18F/git-seekret#18]
When git-seekrets is installed and you're working with a git repo that has submodules, `git commit` will give you a confusing error message about `Object not found - no match for id ($hash_string)` when you attempt to update and commit the submodule changes.

**Comment by alain-hoang** [comment]
`git-seekret` does not have features for working with submodules at this time.  This would require major effort in order to support scanning of submodules.

**Comment by LinuxBozo** [comment]
Closing as confirmed fixed by #20 by @rogeruiz 

```
$ git seekret check -s ; echo $?
Object not found - no match for id (772a7b031bd82d5eb73f130afc1760ac8b6fe409)
1

$ curl -s https://raw.githubusercontent.com/18F/laptop/master/seekrets-install | bash -

Installing Seekret
Installing in /usr/local/bin
...
Finished installing Seekret

$ git seekret check -s ; echo $?
0
```
